### PR TITLE
Remove swift migration guide from current versions

### PR DESCRIPTION
**Pull request by emdobrin**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by calebk1 — COMMENTED** [review]
Just a few comments!

**Review by calebk1 — COMMENTED** [review]
Just a few comments!

File: src/pages/documentation/current-sdk-versions.md [hunk 1]
```diff
@@ -108,7 +100,7 @@ Adobe Experience Platform Mobile SDK plugin for Flutter supports Flutter **versi
 
 <InlineAlert variant="info" slots="text"/>
 
-Adobe Experience Platform Mobile SDK plugins for Flutter are compatible with [Android](#android) and [iOS](#ios) libraries, at this time.
+Adobe Experience Platform Mobile SDK plugins for Flutter are compatible with latest [Android](#android) and [iOS](#ios) native libraries.
```
[the remark this answers is not in the document]
Comment (emdobrin) [reply src/pages/documentation/current-sdk-versions.md]: I will keep "... the Android and iOS ..."

**Review by calebk1 — APPROVED** [review]
LGTM!

**Pull request merged**